src/GitHubUtil.js

```diff
@@ -62,6 +62,64 @@ async function paginatedGet(githubMethod, params) {
     return result;
 }
 
+// Returns recent githubMethod API call entries.
```

Comment:
We need to specify which entries are returned (e.g., the first N, the last N, etc.). Something like this might work:


```suggestion
// Returns the last/latest githubMethod API call entries, in that call pagination order.
```